apps/sms/views/shared/js/contacts.js

```diff
@@ -246,13 +246,13 @@
 
     findExact: function contacts_findExact(filterValue) {
       return this.findBy({
-        filterBy: ['givenName', 'familyName'],
+        filterBy: ['tel', 'givenName', 'familyName'],
         filterOp: 'contains',
```

Comment:
Yup, it makes sense. i will try the code u suggested above and will get back to you with results.